test/plugins/metrics.test.js

```diff
@@ -103,6 +106,7 @@ describe('request metrics plugin', function() {
     it('should return metrics with pre error', function(done) {
         SERVER.on('uncaughtException', function(req, res, route, err) {
             assert.ok(err);
+            res.send(err);
```

Comment:
Is there any exception thrown in this test? It doesn't seem so, it seems there's only an error being passed to a handler's `next` callback? If so, why does the server emit `'uncaughtException'`?

Also, why do we need to `res.send()` now when we wouldn't need to do it before?

Reply:
> f you listen to this event, you must send a response to the client. This behavior is different from the standard error events. If you do not listen to this event, restify’s default behavior is to call res.send() with the error that was thrown. - docs

So it should have been there

Reply:
yes